### DISCUSSION TITLE
Is it possible to get an error message when a relationship in richText is missing?

### DISCUSSION THREAD

**Discussion by Stupidism** [payloadcms/payload#1084] [Feature Requests & Ideas]
When a related entry is deleted, the article itself throws no error message.

```
{
  "data": {
    "Articles": {
      "docs": [
        {
          "slug": "test-missing-relationship",
          "content": [
            {
              "richText": [
                {
                  "children": [
                    {
                      "text": ""
                    }
                  ],
                  "type": "relationship",
                  "value": null,
                  "relationTo": "cards"
                },
                {
                  "children": [
                    {
                      "text": ""
                    }
                  ]
                }
              ],
              "id": "630f06db9a7d2ef790682170",
              "blockName": null,
              "blockType": "content"
            }
          ]
        }
      ]
    }
  }
}
```

**Comment by jmikrut** [comment]
You'd need to build this with custom hooks, but there have been a few people asking for this recently and I believe it could be handled dynamically for sure. However, I'm not sure that throwing an error on `read` of a document with an existing reference to the deleted document would be ideal - rather, you'd probably just want some way of getting a list of documents that reference the document to delete _**before**_ you delete it, so that you can go back and then clean up old references.

Does that make sense? If you build this, we'd love to see it! Otherwise I'm sure this will be something that appears at some point!
